healpy/src/_query_disc.pyx

```diff
@@ -7,7 +7,7 @@ from libcpp.vector cimport vector
 cimport cython
 
 from _common cimport int64, pointing, rangeset, vec3, Healpix_Ordering_Scheme, RING, NEST, SET_NSIDE, T_Healpix_Base
-from _pixelfunc import isnsideok
+from ._pixelfunc import isnsideok
```

Comment:
I'm not sure if it actually matters, but note that this would be more properly `.pixelfunc`, not `._pixelfunc`.